taichi/transforms/lower_ast.cpp

```diff
@@ -101,9 +103,7 @@ class LowerAST : public IRVisitor {
   }
 
   void visit(FrontendBreakStmt *stmt) override {
-    TI_ASSERT_INFO(
-        capturing_loop->is<WhileStmt>(),
-        "The loop capturing 'break' must be a while loop instead of for loop.");
+    TI_ASSERT(capturing_loop->is<WhileStmt>());
```

Comment:
I'm slightly confused - now that we support `break` in certain for loops, why do we still need this assertion?

Reply:
`error: no member named 'find' in 'std::vector<taichi::lang::Stmt *, std::allocator<taichi::lang::Stmt *> >'`

Reply:
Yes, `as<T>()` will do that anyway!